src/linter_options.cc

```diff
@@ -0,0 +1,59 @@
+//
```

Comment:
Please add unit tests for linter_options lib.

Reply:
The only part that can change linter options is NolintParser. So, I was planing to do LinterOptions test along with NolintParser test. (because only reasonable check in LinterOptions is Activity, which is directly related to NolintParser). Now, I made necessary changes to add unit test to execute linter, and we have a simple test. We will have more in future.